python/chronos/test/bigdl/chronos/forecaster/test_lstm_forecaster.py

```diff
@@ -60,6 +60,26 @@ def get_x_y(num_samples):
         return train_data, val_data, test_data
 
 
+def create_tsdataset(roll=True):
+    from bigdl.chronos.data import TSDataset
+    import pandas as pd
+    timeseries = pd.date_range(start='2020-01-01', freq='D', periods=1000)
+    df = pd.DataFrame(np.random.rand(1000, 2),
+                      columns=['value1', 'value2'],
+                      index=timeseries,
+                      dtype=np.float32)
```

Comment:
is there any error if not claiming dtype=np.float32, if yes, where (raise an issue)